lib/Client.php

```diff
@@ -49,7 +49,7 @@ public function getDefaultTimeout()
 
     /**
      * @param RequestInterface $apiRequest
-     * @return \stdClass
+     * @return mixed
```

Comment:
Not everytime this method return an stdClass. It could be an stdClass or an array.

For example: The `PayableHandler` have two method, `get` and `getList`. On pagarme api, the `get` return and object, which is an stdClass, but on the `getList` it return an array.

Maybe mixed it's too broad, but the correct return type should be stdClass | array